crates/ruff_python_ast/src/token_kind.rs

```diff
@@ -0,0 +1,455 @@
+use rustpython_parser::Tok;
+
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
+pub enum TokenKind {
+    /// Token value for a name, commonly known as an identifier.
```

Comment:
Were these comments taken from RustPython?

Reply:
Yep. I copied all variants from `RustPython` directly and removed any associated data.